fortls/parse_fortran.py

```diff
@@ -67,7 +67,7 @@
 DQ_STRING_REGEX = re.compile(r'\"[^\"]*\"', re.I)
 LINE_LABEL_REGEX = re.compile(r'[ ]*([0-9]+)[ ]+', re.I)
 #
-FIXED_COMMENT_LINE_MATCH = re.compile(r'(!|c|d|\*)', re.I)
+FIXED_COMMENT_LINE_MATCH = re.compile(r'^(!|c|d|\*)', re.I)
```

Comment:
Can you remove the added `^` from this regex? Just for consistency with the other patterns. It isn't strictly necessary for your case anyways, since the `match()` method only matches the beginning of the string.

Reply:
done, should be good to go once CI runs